content/en/docs/resources/_index.md

```diff
@@ -9,21 +9,18 @@ aliases:
   - /docs/resources/
 ---
 
-# Presentations
-We have developed a presentation to help you introduce Jenkins X to your team, we will be adding more soon! Right from this site, you can present.
+## Presentations
 
-<iframe src="https://gitpitch.com/sharepointoscar/presentations/master?p=talks/intro-to-jenkins-x#/" frameborder="0" width="1280" height="749"  webkitallowfullscreen="true"></iframe>
+We have developed a presentation to help you introduce Jenkins X to your team, we will be adding more soon! Right from this site, you can present.
 
+<iframe src="https://gitpitch.com/sharepointoscar/presentations/master?p=talks/intro-to-jenkins-x#/" frameborder="0" scrolling="no" allowtransparency="true" allowfullscreen="true" id="emb_5jtc6n" height="401" width="600" style="border: none; position: relative; visibility: visible; display: block; margin: 0px; padding: 0px; max-width: 100%; min-width: 200px;"></iframe>
```

Comment:
Not related to this PR, but the gitpitch slides (page 11) say `jx` runs on aks, eks and gke, may be remove aks?

Reply:
that would be a something for @sharepointoscar since this is his presentation

Reply:
@ankitm123 I updated the deck, removed Azure since it is WIP.